ulib/experimental/Steel.Channel.Simplex.fst

```diff
@@ -368,7 +369,9 @@ let rec send (#p:prot) (c:chan p) (#next:prot{more next}) (x:msg_t next)
   = let v = send_receive_prelude c in //matching v as vs,vr fails
     if (fst v).chan_ctr = (snd v).chan_ctr
     then (
-      rewrite_context();
+      change_slprop (chan_inv_cond (fst v) (snd v))
+                    (pure (fst v == snd v))
+                    (fun _ -> ());
```

Comment:
Similar comment here, any diagnosis of requiring these changes?

Reply:
Same comment as above. I didn't inspect the new constraints in detail, but I didn't expect these rewrite_context to be robust.